src/test/kotlin/nl/info/zac/admin/ZaaktypeConfigurationServiceTest.kt

```diff
@@ -0,0 +1,148 @@
+/*
+ * SPDX-FileCopyrightText: 2025 INFO.nl
+ * SPDX-License-Identifier: EUPL-1.2+
+ */
+package nl.info.zac.admin
+
+import io.kotest.core.spec.style.BehaviorSpec
+import io.mockk.every
+import io.mockk.just
+import io.mockk.mockk
+import io.mockk.runs
+import io.mockk.verify
+import jakarta.persistence.EntityManager
+import jakarta.persistence.criteria.CriteriaQuery
+import nl.info.client.zgw.ztc.ZtcClientService
+import nl.info.client.zgw.ztc.model.createZaakType
+import nl.info.zac.admin.model.ZaaktypeConfiguration
+import nl.info.zac.admin.model.createZaaktypeCmmnConfiguration
+import nl.info.zac.flowable.bpmn.model.createZaaktypeBpmnConfiguration
+import java.net.URI
+import java.util.UUID
+
+class ZaaktypeConfigurationServiceTest : BehaviorSpec({
+    val zaaktypeUri = URI("https://example.com/zaaktype/${UUID.randomUUID()}")
+
+    val entityManager = mockk<EntityManager>()
+    val ztcClientService = mockk<ZtcClientService>()
+    val zaaktypeCmmnConfigurationBeheerService = mockk<ZaaktypeCmmnConfigurationBeheerService>()
+    val zaaktypeBpmnConfigurationBeheerService = mockk<ZaaktypeBpmnConfigurationBeheerService>()
+    val zaaktypeConfigurationService = ZaaktypeConfigurationService(
+        entityManager,
+        ztcClientService,
+        zaaktypeCmmnConfigurationBeheerService,
+        zaaktypeBpmnConfigurationBeheerService
+    )
+
+    Given("a concept zaaktype event") {
+        val zaaktype = createZaakType(concept = true)
+
+        every { ztcClientService.clearZaaktypeCache() } returns ""
+        every { ztcClientService.readZaaktype(zaaktypeUri) } returns zaaktype
+
+        When("updating zaakafhandel parameters") {
+            zaaktypeConfigurationService.updateZaakafhandelParameters(zaaktypeUri)
```

Comment:
The naming still very much confuses me.. Why is this function not named `updateZaaktypeConfiguration`? People who see this now start thinking: is a zaakafhandelparameters the same thing as a zaaktype configuration or not?

I suggest to finish what you started: in our code just use `zaaktype configuration` everywhere and treat it as the EN translation of `zaakafhandelparameters`.